src/Templates.js

```diff
@@ -36,16 +35,16 @@ let API_OBJECT_ID = {};
 })();
 
 function UNIVERSAL_GETTER_ID(variableName) {
-  if (REWIRED_DATA_IDENTIFIER === undefined || REWIRED_DATA_IDENTIFIER[variableName] === undefined) {
-    return ORIGINAL_VARIABLE_ACCESSOR_IDENTIFIER(variableName);
-  } else {
-    var value = REWIRED_DATA_IDENTIFIER[variableName];
-    if (value === INTENTIONAL_UNDEFINED) {
-      return undefined;
-    } else {
-      return value;
-    }
-  }
+	if (REWIRED_DATA_IDENTIFIER === undefined || REWIRED_DATA_IDENTIFIER[variableName] === undefined) {
+		return ORIGINAL_VARIABLE_ACCESSOR_IDENTIFIER(variableName);
+	} else {
+		var value = REWIRED_DATA_IDENTIFIER[variableName];
```

Comment:
`const`?

Reply:
I just fixed the tabs to be consistent.

Reply:
All of these `vars` and so on are from original code, so should probably not be changed in this PR.